assets/app/views/builds.html

```diff
@@ -8,21 +8,86 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>
-      <div style="margin-bottom: 10px;" ng-repeat="build in builds">
-        <h3>{{build.metadata.labels.buildconfig}} <span class="small">({{build.metadata.name}})</span></h3>
-        <div>Created: <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></div>            
-        <div>Status: {{build.status}}</div>
-        <div>Build strategy: {{build.parameters.strategy.type}}</div>
-        <div ng-if="build.parameters.strategy.type == 'STI'">
-          Builder image: {{build.parameters.strategy.stiStrategy.builderImage}}
+      <div ng-repeat="(buildConfigName, buildConfig) in buildConfigs">
+        <h2>{{buildConfigName}}</h2>
+        <div>Build strategy: {{buildConfig.parameters.strategy.type}}</div>
+        <div ng-if="buildConfig.parameters.strategy.type == 'STI'">
+          Builder image: {{buildConfig.parameters.strategy.stiStrategy.builderImage}}
         </div>
-        <div ng-if="build.parameters.source">
-          <div ng-if="build.parameters.source.type == 'Git'">Source repo: {{build.parameters.source.git.uri}}</div>
-          <!-- TODO add git ref details -->
+        <div ng-if="buildConfig.parameters.source">
+          <div ng-if="buildConfig.parameters.source.type == 'Git'">Source repo: {{buildConfig.parameters.source.git.uri}}</div>
         </div>
-        <div>Output image: {{build.parameters.output.imageTag}}</div>   
-        <div>Output image registry: {{build.parameters.output.registry}}</div>   
-      </div>      
+        <div ng-switch="buildConfig.parameters.output.to.kind">
```

Comment:
@csrwng can you send me a sample template that has a build that would produce dockerImageReference in the API output, all the examples under the sample-app do not return dockerImageReference